packages/config/src/configs/moonbeam.ts

```diff
@@ -724,6 +725,21 @@ export const moonbeamConfig = new ChainConfig({
         balance: BalanceBuilder().substrate().system().account(),
       },
     }),
+    new AssetConfig({
+      asset: bncs,
+      balance: BalanceBuilder().evm().erc20(),
+      contract: ContractBuilder().Xtokens().transfer(),
+      destination: bifrostPolkadot,
+      destinationFee: {
+        amount: 0.00000001,
```

Comment:
```suggestion
        amount: 0.0001,
```
incorrect fee, would cause transaction failure